packages/world/src/timeToGrow.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+/* Autogenerated file. Do not edit manually. */
+
+import { ObjectTypeId } from "./ObjectTypeId.sol";
+import { ObjectTypes } from "./ObjectTypes.sol";
+
+function timeToGrow(ObjectTypeId objectTypeId) pure returns (uint128) {
+  if (objectTypeId == ObjectTypes.WheatSeed) {
```

Comment:
Why doenst this have the tree seeds?

Reply:
also can we make this 15 minutes? and then use scripts to speed up time when doing dev locally